### PR TITLE
Feat/fix isidore saint translate Spanish.

### PR DESCRIPTION
**Pull request by miguelms95**
Is "San **Isidoro** de Sevilla, obispo y doctor de la Iglesia" not "**Isidro**"
https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=5

### CODE REVIEW

**Comment by miguelms95** [comment]
@emagnier please check when possible :)

File: lib/locales/es.ts [hunk 1]
```diff
@@ -357,7 +357,7 @@ export const locale: Locale = {
     innocent_xi_pope: 'Beato Inocencio XI, papa',
     irenaeus_of_lyon_bishop: 'San Ireneo de Lyon, obispo, mártir y doctor de la Iglesia',
     isabelle_of_france_virgin: 'Beata Isabel de Francia, virgen',
-    isidore_of_seville_bishop: 'San Isidro de Sevilla, obispo y doctor de la Iglesia',
+    isidore_of_seville_bishop: 'San Isidoro de Sevilla, obispo y doctor de la Iglesia', // https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=5
```
Comment (tukusejssirs) [inline lib/locales/es.ts]:
```suggestion
    isidore_of_seville_bishop: 'San Isidoro, obispo y doctor de la Iglesia', // src: https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=5
```

**Review by tukusejssirs — CHANGES_REQUESTED** [review]
@miguelms95, thanks for this PR! :pray:

The Proper does not use `de Sevilla` for the saint, thus it should be removed from romcal. After changing this, this PR looks all good to me.

**Comment by miguelms95** [comment]
Done! @tukusejssirs 👍🏼

**Comment by tukusejssirs** [comment]
Anyway, could you fix the commit messages please? See the [CI log](https://github.com/romcal/romcal/actions/runs/8845937411/job/24290825445?pr=530#step:6:1).

And you should rebase your changes on top of `dev`. Maybe the easiest way is to:

- back up `es.ts`;
- change the `HEAD` of your feature branch (`feat/fix-isidore-saint`) to point to that of `dev` (`git fetch upstream && git reset --hard upstream/dev`; this presumes you have added an `upstream` remote to Git by `git remote add upstream https://github.com/romcal/romcal.git && git remote set-url --push upstream no_push`);
- restore your changes;
- create a new commit (e.g. `fix(es): fix the localisation of isidore_of_seville_bishop`);
- force-push your changes.

File: lib/locales/es.ts (hunk 1 above)
Comment (tukusejssirs) [reply lib/locales/es.ts]: Nitpick: @miguelms95, could you also add `src: ` before the source link please? :pray:

**Comment by miguelms95** [comment]
Done! ✅ @tukusejssirs thanks for the detailed reply. Now there's only a commit haha 👍🏼

**Review by tukusejssirs — APPROVED** [review]

**Review by emagnier — APPROVED** [review]
Thank you all for the fix and the review 🚀

**Pull request merged**